doc/source/drivers/vector/parquet.rst

```diff
@@ -0,0 +1,69 @@
+.. _vector.parquet:
+
+(Geo)Parquet
+============
+
+.. shortname:: Parquet
+
+.. build_dependencies:: Parquet component of the Apache Arrow C++ library
+
+From https://databricks.com/glossary/what-is-parquet:
+"Apache Parquet is an open source, column-oriented data file format designed
+for efficient data storage and retrieval. It provides efficient data compression
+and encoding schemes with enhanced performance to handle complex data in bulk.
+Apache Parquet is designed to be a common interchange format for both batch and interactive workloads."
+
+This driver also supports geometry columns using the GeoParquet specification.
+
+.. note:: The driver should be considered experimental as the GeoParquet specification is not finalized yet.
+
+
+Driver capabilities
+-------------------
+
+.. supports_create::
+
+.. supports_georeferencing::
+
+.. supports_virtualio::
+
+
+Creation issues
+---------------
+
+The driver supports creating only a single layer in a dataset.
+
+Layer creation options
+----------------------
+
+- **COMPRESSION=string**: Compression method. Can be one of ``NONE``, ``SNAPPY``,
+  ``GZIP``, ``BROTLI``, ``ZSTD``, ``LZ4``, ``BZ2``, ``LZ4_HADOOP``. Available
+  values depend on how the Parquet library was compiled.
+  Defaults to SNAPPY when available, otherwise NONE.
+
+- **GEOMETRY_ENCODING=WKB/WKT/GEOARROW**: Geometry encoding. Defaults to WKB.
+  Other encodings (WKB and WKT) are *not* allowed by the GeoParquet
+  specification, but are handled as an extension, for symmetry with the Arrow
+  driver.
+
```

Comment:
Something does not match. Is the default perhaps GEOARROW?

Reply:
> Is the default perhaps GEOARROW?

no, it is WKB. This was a copy&paste error from the doc of the Arrow driver. I've pushed a commit to rectify to: "Defaults to WKB.  Other encodings (WKT and GEOARROW) are *not* allowed" ..."